conan/cli/formatters/graph/info_graph_html.py

```diff
@@ -69,12 +69,31 @@
                                    '</ul>'
                     }{%- if not loop.last %},{% endif %}
                 {%- endfor %}
-            ]);
-            var edges = new vis.DataSet([
+            ]
+            var edges = [
                 {%- for src, dst in graph.edges %}
                     { from: {{ src.id }}, to: {{ dst.id }} }{%- if not loop.last %},{% endif %}
                 {%- endfor %}
-            ]);
+            ]
+
+            {% if error is not none and error["type"] == "conflict" %}
```

Comment:
I would try to also highlight in big-red the other part of the conflict. In case it is a relatively large graph it will not be evident to see it. 2 big red things easy to identify would probably be the best.

Reply:
Done! see my comment for the new look